pkgs/development/python-modules/tftpy/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "tftpy";
+  version = "0.8.2";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-4dGmgO/YjroXazURdYRCUwZzkqmw+LgViOP/K557u1s=";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
```

```suggestion
  nativeBuildInputs = [ setuptools ];
```
`setuptools` is not required for propagation.